examples/gno.land/p/pflow/metamodel/gno.mod

```diff
@@ -0,0 +1 @@
+module gno.land/p/pflow/metamodel
```

Comment:
Mhh, I don't know if we want to have "org names" as namespaces at launch, maybe I'd suggest putting this in r/stackdump/pflow/metamodel